src/reducers/pause.js

```diff
@@ -60,27 +61,29 @@ export type PauseState = {
   loadedObjects: Object,
   shouldPauseOnExceptions: boolean,
   shouldPauseOnCaughtExceptions: boolean,
-  canRewind: boolean,
-  debuggeeUrl: string,
   command: Command,
   previousLocation: ?MappedLocation,
   skipPausing: boolean
 };
 
-export const createPauseState = (): PauseState => ({
-  ...emptyPauseState,
-  extra: {},
-  isWaitingOnBreak: false,
-  shouldPauseOnExceptions: prefs.pauseOnExceptions,
-  shouldPauseOnCaughtExceptions: prefs.pauseOnCaughtExceptions,
+// Pause state describing all threads.
+export type PauseState = {
+  mainThread: string,
+  currentThread: string,
+  debuggeeUrl: string,
+  canRewind: boolean,
+  threads: Object
```

Comment:
can we type this?

Reply:
I think so, it looks like this can be { [string]: PauseState }